app/api/api_v1/endpoints/student_router.py

```diff
@@ -33,17 +33,17 @@ async def list_students(
     students = await StudentService(db).list_students()
     return students
 
-@router.post("/students", response_model=StudentSchema)
-async def create_student_with_autogen_password(
-    *,
-    db: Session = Depends(get_db),
-    perm: None = Depends(PermissionDependency(StudentCreatePermission)),
-    student_body: CreateStudentBody
-):
-    student = await StudentService(db).create_student(
-        **student_body.dict()
-    )
-    return student
+# @router.post("/students", response_model=StudentSchema)
```

Comment:
Is this temporary in the PR? If you're planning to commit this commented out, can you add another comment above saying why it's commented out?